MyCryptoHQ/MyCrypto
# Add Check TX Button to Transaction Notification

Closes #1161.

### Description

Adds a button to the transaction succeeded notification that links to /tx-status with the hash pre-filled.

### Changes

* Add a check tx button to TransactionSucceeded.tsx
* Fix the notification to work with custom networks without block explorers
* Fix alert `btn` styles
    * I just decided to override all button styles, so buttons in alerts look uniform.
* Fix LogOutPrompt to carry over the entire url, not just the path
* Make /tx-status take in the ?txHash query search parameter
* Add some stricter typings to the broadcast helpers saga


### Steps to Test

1. Run the app on the `check-tx-notif` branch
2. Send a transaction
3. Click on "Check TX Status"
4.Say yes to the logout confirmation
5. Confirm that you're seeing the correct transaction

### Screenshots

<img width="1440" alt="screen shot 2018-03-01 at 1 45 17 pm" src="https://user-images.githubusercontent.com/649992/36865269-0667c162-1d5d-11e8-89c3-1577381bbf57.png">
